src/helpers/VSACHelper.ts

```diff
@@ -0,0 +1,28 @@
+const VSAC_REXEG = /http:\/\/cts\.nlm\.nih\.gov.*ValueSet/;
```

Comment:
is REXEG instead of REGEX intentional? What is REXEG?

Reply:
incredible